### PR TITLE
Add psiclass recipe

### PR DESCRIPTION
**Pull request by mourisl**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by mourisl** [comment]
@bioconda/core Could you please merge the recipe of the transcriptome assembler PsiCLASS to bioconda? Thank you.

File: recipes/psiclass/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - samtools 0.1.19.*
+    - zlib
+  run:
+    - zlib
```
Comment (bgruening) [inline recipes/psiclass/meta.yaml:20]: zlib can be removed from the run section

File: recipes/psiclass/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - trust-splice 2>&1 | cat > /dev/null
+    - subexon-info 2>&1 | cat > /dev/null
+    - combine-subexons 2>&1 | cat > /dev/null
+    - vote-transcripts 2>&1 | cat > /dev/null 
+    - add-genename 2>&1 | cat > /dev/null
+    - which psiclass
+
+about:
+  home: https://github.com/splicebox/PsiCLASS
+  license: GPLv3
```
Comment (bgruening) [inline recipes/psiclass/meta.yaml:36]: Can you pleae use this identifier: https://spdx.org/licenses/GPL-3.0.html and also include `licnese:`

File: recipes/psiclass/meta.yaml (hunk 1 above)
Comment (mourisl) [reply recipes/psiclass/meta.yaml]: I've removed the zlib from the file.

File: recipes/psiclass/meta.yaml (hunk 2 above)
Comment (mourisl) [reply recipes/psiclass/meta.yaml]: I changed the license to "license: GNU General Public License v3.0 only". Is it fine? Thank you!

**Comment by bgruening** [comment]
Can you please add a `license_file` as well?

**Comment by mourisl** [comment]
I've added the LICENSE file. Please check. Thank you!

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**